i18n/locales/source/dashboard/marketing_announcements.json

```diff
@@ -94,6 +94,11 @@
       "title": "AI 101 for Teachers",
       "body": "Discover the groundbreaking world of Artificial Intelligence (AI) and its transformative potential in education with our comprehensive online learning series for teachers.",
       "buttonText": "Learn more"
+    },
+    "hoc-2023": {
+      "title": "Hour of Code: Creativity with AI",
+      "body": "Join millions across the globe in organizing an hour of coding, with or without AI and learning how AI works. Anyone, anywhere can do it. No experience needed.",
+      "buttonText": "Learn more"
```

Comment:
This file is supposed to allow for translations based off of [this PR](https://github.com/code-dot-org/code-dot-org/pull/53959) — @mgc1194 can you take a look here and let me know if there's anything else I need to do to make sure these get translated?